VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
+1 for version in Makefile. There is no benefit at all in external version file.